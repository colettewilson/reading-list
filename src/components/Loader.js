import React from "react"
import styles from "./Loader.module.css"

const Loader = () => {
  return (
    <div className={styles.loader} data-testid="loader">
      <span className={styles.loaderItem}></span>
      <span className={styles.loaderItem}></span>
      <span className={styles.loaderItem}></span>
      <p>Loading</p>
    </div>
  )
}

export default Loader
