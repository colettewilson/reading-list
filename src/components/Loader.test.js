import React from "react"
import { render, cleanup } from "@testing-library/react"
import Loader from "./Loader"

afterEach(cleanup)

describe("Loader Component", () => {
  test("should match snapshot", () => {
    const { container } = render(<Loader />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
