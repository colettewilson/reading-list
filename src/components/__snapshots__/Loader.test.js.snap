// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should match snapshot 1`] = `
<div
  class="loader"
>
  <span
    class="loaderItem"
  />
  <span
    class="loaderItem"
  />
  <span
    class="loaderItem"
  />
  <p>
    Loading
  </p>
</div>
`;
